File: WEB-INF/classes/IRepository.java

import java.util.ArrayList;

public interface IRepository {
    void init(String connectString);
    void close();
    void insert(AClass entry);
    void update(AClass entry, String pkey, String values);
    void delete(String tableType, String criteria);
    ArrayList<AClass> select(String tableType, String criteria);
}

File: QuizAppTest/Utilities/IRepository.cs
using System.Formats.Asn1;

namespace QuizApp.Utilities
{
    public interface IRepository
    {
        void init(DatabaseUtil databaseUtil);
        void close();
        void insert(AClass entry);
        void update(AClass entry, string pkey, string values);
        void delete(string tableType, string criteria);
        List<AClass> select(string tableType, string criteria);
    }
}
